shop.go

```diff
@@ -39,20 +58,23 @@ func NewShop(shop string, accessToken string) Shop {
 	This constructor automatically uses the URL passed to it to communicate with the store.
 	If you would like to use an auto store URL use the `NewShop` constructor instread.
 	Example:
-	shop := shopify.NewCustomShop("https://my-shop-domain.com/foo/bar", "shppy_21u92h2184ho912h29r01")
+	shop := shopify.NewCustomShop("https://my-shop-domain.com/foo/bar", "shppy_21u92h2184ho912h29r01", httpshopify.IsPlus)
 	shippedOrders, err := shop.Orders().List(shopify.OrderQuery{FulfillmentStatus:"shipped"})
 	For the full shopify admin REST API documentation see https://shopify.dev/docs/admin-api/rest/reference
 */
-func NewCustomShop(url string, accessToken string) Shop {
-	client := http.NewClient(http.RequestHeaders{
-		{
-			Name:  "X-Shopify-Access-Token",
-			Value: accessToken,
-		}, {
-			Name:  "Content-Type",
-			Value: "application/json",
-		},
-	})
+func NewCustomShop(url string, accessToken string, isPlus bool) Shop {
```

Comment:
For a fully custom shop should we allow them to provide there own limiting options? Even as far to say they can provide there own client...

Reply:
yeah this is an interest point, the original idea behind this thing was just to allow the user of the package to set their own fullpath to the shop, but then I wanted to be able to specify if the store was plus or not, now I'm wondering if we need more powerful constructors. Maybe pushing as much as possible into options?

Reply:
I think so, but I would leave it as a future issue. As we cover the most use-cases with our two implementations, if we get a need for a third then it would be worth refactoring.